packages/dds/test-dds-utils/src/ddsFuzzHarness.ts

```diff
@@ -1088,6 +1131,10 @@ function isInternalOptions(options: DDSFuzzSuiteOptions): options is InternalOpt
 	return options.only instanceof Set && options.skip instanceof Set;
 }
 
+// eslint-disable-next-line jsdoc/require-description
```

Comment:
Is this rule disable needed? It doesn't look like we need it elsewhere in this package.

Reply:
![image](https://github.com/microsoft/FluidFramework/assets/111468570/b36a4005-abc7-477a-9fee-d36eeef44024)

The disable is needed as the linter is complaining about the missing JSDoc. This is same for the `test-dds-utils/gcTestRunner.ts`.

Reply:
Okay, just wanted to double check. Some of our packages have this rule enabled, and some don't. Looks good!